### PR TITLE
Allow usage of async/await

### PR DESCRIPTION
**Pull request by topaxi**
Every browser with SW support, supports async/await too.

This PR removes `regenerator` from the babel options and updates `broccoli-rollup` and `broccoli-uglify-sourcemap` to be able to parse async/await syntax.

### CODE REVIEW

File: lib/get-babel-options.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ function getBabelOptions(addonContext) {
     options.blacklist.push('es6.modules');
   }
 
+  // All browsers which support service workers support async/await
+  options.blacklist.push('regenerator');
```
Comment (martndemus) [inline lib/get-babel-options.js]: Can you test async await in a service worker registration file on an unsupported browser? See if it doesn't crash and burn on syntax errors?

**Comment by martndemus** [comment]
Looks great! But does it not crash on a registration if you use async/await in an unsupported browser?

**Comment by bcardarella** [comment]
If it does I'm in favor of a progressive enhancement path rather than bringing in regenerator. The footprint of that plugin is not small.

**Comment by topaxi** [comment]
This will defintely "crash and burn" if one uses async/await in sw-registration code in browsers without SW support. It would crash anyways today as we'd need to import regenerator explicitly (I'm not sure if ember-maybe-import-regenerator works here, never tried).

So this is only a problem if one uses async AND ships code for non-SW browser in registration code (which might not be that uncommon? I don't know).

Maybe implement this as a flag? Or the ability to customize babel options for ember-service-worker?

```js
// ...
module.exports = function(defaults) {
  let app = new EmberApp(defaults, {
    'ember-service-worker': {
      babel: { blacklist: ['regenerator'] },
    },
  });
  // ...
};
```

**Comment by martndemus** [comment]
I like that idea, to supplement the babel options if possible.

**Pull request closed without merging**